docs/deployment.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Deployment and CI/CD Process
+---
+
+# Deployment and CI/CD Process
+
+## Overview
+
+DEV relies on GitHub and Travis to deploy continuously to Heroku. If a Pull Request is merged without the "ci skip" in its title, it will be automatically deployed to production once the build steps complete successfully. The process currently takes about 20 minutes to complete and will need a few additional minutes before the change goes live.
```

Comment:
Don't forget we need to update this if/when your other PR goes out 💃

Reply:
Yup! I'll merge this then update it on the other PR.